keymapviz/keyboards/lily58/__init__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+keymap_keyword = '(?:LAYOUT|LAYOUT_kc|LAYOUT_split_3x6_7_4)'
```

Comment:
@daanhorn 

I checked the keymap.c in [this directory](https://github.com/qmk/qmk_firmware/tree/master/keyboards/lily58/keymaps), but I could not find out which `keymap.c` uses `LAYOUT_split_3x6_7_4`. I would like to remove it if we are not using it. If you know of a `keymap.c` that uses `LAYOUT_split_3x6_7_4`, please let me know. It will be easier to maintain.

And [this](https://github.com/yskoht/keymapviz/blob/92bbe164ff4ab3f85515f56a0f088b39e218ef92/keymapviz/keyboards/kyria/__init__.py#L3) too :)

Reply:
LAYOUT_kc is something I copied. Should probably be removed on both.